kafka/pom.xml

```diff
@@ -23,15 +23,15 @@
     <parent>
         <groupId>io.cloudevents</groupId>
         <artifactId>cloudevents-parent</artifactId>
-        <version>2.4.0-SNAPSHOT</version>
+        <version>2.5.0-SNAPSHOT</version>
     </parent>
 
     <artifactId>cloudevents-kafka</artifactId>
     <name>CloudEvents - Kafka Binding</name>
     <packaging>jar</packaging>
 
     <properties>
-        <kafka.version>2.5.0</kafka.version>
+        <kafka.version>3.0.0</kafka.version>
```

Comment:
There are some unrelated changes like this one, would you mind rebasing?

Reply:
@pierDipi - That's for your patience - rebased & pushed